conf/messages

```diff
@@ -1329,9 +1329,9 @@ itemDensity.change.hidden = density
 itemDensity.suffix = kg/m<sup>3</sup>
 itemDensity.hint = Enter the density, at 15 degrees Celsius per kilogram per cubic metre.
 
-itemsPackagingAddToList.title.singular = You have added 1 packaging type for item {0}
+itemsPackagingAddToList.title.singular = You have added 1 packaging type for item {1}
```

Comment:
Shouldn't this be:
```suggestion
itemsPackagingAddToList.title.singular = You have added 1 packaging type for item {0}
```

Reply:
Just ran the code and it runs as intended (showing the correct item number), it does look strange when viewing this solely from a message point of view

Reply:
I'll add a comment explaining why this is `{1}` not `{0}` - took me a minute to figure out why too 😅